.travis.yml

```diff
@@ -13,7 +13,11 @@ install:
 script:
   - cd test
   - pip freeze  # so to help eventual debug: know what exact versions are in use can be rather useful.
-  - nosetests -xv --process-restartworker --processes=1 --process-timeout=999999999  --with-cov --cov=alignak
+  - which nosetests
+  - nosetests -V
```

Comment:
Well readibility purpose IMO. Add some which travis and so is As ugly as letting print statement in python or letty -x in bash.